webtau-http-groovy/src/test/groovy/org/testingisdocumenting/webtau/http/HttpGroovyTest.groovy

```diff
@@ -539,6 +539,30 @@ class HttpGroovyTest extends HttpTestBase {
         body.get("list[2]").traceableValue.checkLevel.should == CheckLevel.None
     }
 
+    @Test
```

Comment:
add an example for the alias (now that you put work into generating it)?

Reply:
what do you mean by generating it?